src/Logger.php

```diff
@@ -6,20 +6,28 @@
 
 class Logger {
 	static bool $exitOnError = true;
+	static bool $verbose = false;
 
 	protected static function log(LoggerLevel $level, string $context, string $text): void {
 		print(self::format($level, $context, $text));
 	}
 
 	protected static function format(LoggerLevel $level, string $context, string $text): string {
 		$colorCode = match ($level) {
-			LoggerLevel::Info => "",
+			LoggerLevel::Debug => "\e[95m",
```

Comment:
🙈

Reply:
Do you prefer no or a different color?

Reply:
I would keep it default since info is colored now,
but anyway, continue